src/components/Table.tsx

```diff
@@ -0,0 +1,49 @@
+import React, { Component } from 'react';
+import { computed } from 'mobx';
+import { observer } from 'mobx-react';
+import autoBindMethods from 'class-autobind-decorator';
+
+import * as Antd from 'antd';
+
+import { fieldSetsToColumns } from '../utilities/common';
+import { IModel, ISharedComponentProps } from '../props';
+import { ColumnProps } from '../../node_modules/antd/lib/table/interface';
+
+export interface ITableProps extends ISharedComponentProps {
+  model: IModel[];
+}
+
+@autoBindMethods
+@observer
+class Table extends Component<ITableProps> {
+  @computed
+  private get columns (): Array<ColumnProps<IModel>> {
+    return fieldSetsToColumns(this.props.fieldSets);
+  }
+
+  @computed
+  private get dataSource (): Array<IModel & { key: string }> {
+    return this.props.model.map((item, idx) => ({
+      key: item.id || idx.toString(),
+      ...item,
+    }));
+  }
+
+  private title () {
+    return this.props.title;
+  }
+
+  public render () {
+    return (
+      <Antd.Table
+        {...this.props}
+        className='ant-table-simple'
```

Comment:
This shouldn't be here, it's registry related. Also, to me `fields-ant` should not define styling in any way.
(My comment above says that you can just target the regular `.ant-table`, does the same job.)

FYI, we should always think about how this can be used. for example, the way it's done now, I'd have no way to set a className or a custom title because it would be overwritten because of the props order.

This would be more adequate:
```
<Antd.Table
    title={this.title}
    {...this.props}
    columns={this.columns}
    dataSource={this.dataSource}
/>
```

Reply:
Title has to come after this.props since title is a string but Table wants a function for title.